app/src/main/java/org/wikipedia/views/ImagePreviewDialog.kt

```diff
@@ -137,6 +134,14 @@ class ImagePreviewDialog : ExtendedBottomSheetDialogFragment(), DialogInterface.
         ViewUtil.loadImageUrlInto(galleryImage, url)
     }
 
+    private fun getClickListenerFor(url: String?): View.OnClickListener {
+        return View.OnClickListener {
```

Comment:
Should it have a `url` null check here? I am seeing some links will open a blank page.

Reply:
I don't think it is null urls. sometimes the response itself doesn't have well-formed ones. However, I have put in a null check.